### DISCUSSION TITLE
It is possible to do the downgrade / jailbreak procedure without HOME button?

### DISCUSSION THREAD

**Discussion by diegopds83** [LukeZGD/Legacy-iOS-Kit#417] [Q&A] [answered]
Hello. The Home button of my iPhone 4 GSM is broken. Still, is it possible to do the downgrade / jailbreak procedure without it?

Thanks!

**Comment by LukeZGD** [comment, marked as answer]
Yes it's still possible, but it will require restoring the device (this wipes data).

To do downgrade/jailbreak without home button, just go to Restore/Downgrade -> DFU IPSW. After the restore finishes, the device is now in DFU mode. Run restore.sh again and go to Restore/Downgrade -> powdersn0w or latest. Start the restore, and enable the jailbreak option
